orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/ExecutionLauncher.java

```diff
@@ -20,25 +20,33 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.netflix.appinfo.InstanceInfo;
 import com.netflix.spinnaker.orca.pipeline.model.Execution;
+import com.netflix.spinnaker.orca.pipeline.persistence.ExecutionRepository;
 import lombok.extern.slf4j.Slf4j;
 
 @Slf4j
 public abstract class ExecutionLauncher<T extends Execution> {
 
   protected final ObjectMapper objectMapper;
   protected final InstanceInfo currentInstance;
+  protected final ExecutionRepository executionRepository;
+
   private final ExecutionRunner runner;
 
   protected ExecutionLauncher(ObjectMapper objectMapper,
                               InstanceInfo currentInstance,
+                              ExecutionRepository executionRepository,
                               ExecutionRunner runner) {
     this.objectMapper = objectMapper;
     this.currentInstance = currentInstance;
+    this.executionRepository = executionRepository;
     this.runner = runner;
   }
 
   public T start(String configJson) throws Exception {
     final T execution = parse(configJson);
+
+    persistExecution(execution);
```

Comment:
This jives with historical behavior where we persisted the skeleton (before stages got blown out) ... the SpringBatchExecutionRunner will re-persist once it has finished building the execution.